pkg/loader/compose/compose.go

```diff
@@ -178,6 +183,50 @@ func loadEnvVars(envars []string) []kobject.EnvVar {
 	return envs
 }
 
+// Convert the Docker Compose v3 volumes to []string (the old way)
+// TODO: Check to see if it's a "bind" or "volume". Ignore for now.
+// See: https://docs.docker.com/compose/compose-file/#long-syntax-2
+func loadV3Volumes(volumes []types.ServiceVolumeConfig) []string {
```

Comment:
@surajssd Also, #626 should be merged in after, improving volumes could be done after this PR. Getting this experimental functionality merged-in would be best.